api/src/test/java/org/cardanofoundation/rosetta/api/construction/service/PayloadsApiTest.java

```diff
@@ -69,13 +73,23 @@ void stakeKeyDeregistrationTest() throws Exception {
   @Test
   void operationsHaveIdentifierTest() {
     ConstructionPayloadsRequest request = ConstructionPayloadsRequest
-            .builder()
-            .operations(List.of(new Operation()))
-            .build();
-    ApiException exception = Assertions.assertThrows(ApiException.class,
-            () -> constructionApiService.constructionPayloadsService(request));
-    Assertions.assertEquals("body[0] should have required property operation_identifier",
-            exception.getError().getDetails().getMessage());
+        .builder()
+        .operations(List.of(Operation.builder()
+            .type("operation")
+            .build()))
+        .networkIdentifier(NetworkIdentifier.builder()
+            .blockchain("cardano")
+            .network("mainnet")
+            .build())
+        .build();
+
+    // Check for not null Identifier is on the level above, at the constructionApi
+    ConstraintViolationException exception = Assertions.assertThrows(
+        ConstraintViolationException.class,
+        () -> constructionApi.constructionPayloads(request));
+
+    Assertions.assertEquals("constructionPayloads.arg0.operations[0].operationIdentifier: must not be null",
```

Comment:
TS implementation has a bit different error:
`An error occurred for request 793894: body.operations[0] should have required property 'operation_identifier'`
However, our check is on the level of the generated class, so changing only one message would not make sense.